gr-blocks/include/gnuradio/blocks/blockinterleaving.h

```diff
@@ -0,0 +1,57 @@
+/* -*- c++ -*- */
+/*
+ * Copyright 2022 Johannes Demel.
+ *
+ * This file is part of GNU Radio
+ *
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ *
+ */
+
+#ifndef INCLUDED_GR_BLOCKS_BLOCKINTERLEAVING_H
+#define INCLUDED_GR_BLOCKS_BLOCKINTERLEAVING_H
+
+#include <gnuradio/blocks/api.h>
+
+
+namespace gr {
+namespace blocks {
```

Comment:
should `blockinterleaving` be in the `kernel` namespace as well?

Reply:
Well. It is a kernel. I didn't think of it. Sounds reasonable. Just to make sure

`gr::blocks::kernel::blockinterleaving` should be correct?

Reply:
that seems like it would be consistent